packages/web/docs/src/content/gateway/authorization-authentication.mdx

```diff
@@ -24,6 +24,12 @@ gateway, and no other entity can execute requests to the subgraph on behalf of t
 
 </Callout>
 
+<Callout type="warning">
```

Comment:
I assume this is here because of the "subgraph authentication"? I feels a bit confusing, I feel like it should be under security only like you did? 

Another option is to leave this section but make it broader, something like: "Looking for Gateway <> Subgraph authentication and request signature? ..."

Reply:
The title says Authorization and Authentication in this section. So people will expect anything auth related is here including HMAC and AWS Sigv4. Like the warning above this about HMAC i added this so people can find the dedicated section about AWS Sigv4. Like i said earlier when we build this docs, it was not a good idea making a section titled as Auth but only mentions about JWT between GW and the consumer.